eva-accession-release/src/main/resources/copy_release_to_ftp.sh

```diff
@@ -0,0 +1,226 @@
+ALL_SPECIES_FOLDER=by_species
+
+if [ $# -ne 4 ]
+then
+  echo -e "\nThis script copies the relevant files of all the species into the FTP folder for the EVA RS release."
+  echo "The user needs to be able to run 'become ftpadmin' to run this script."
+  echo -e "This script needs as parameter 4 folders: input, output, intermediate and unmapped variants."
+  echo -e "Also, recommended to run in LSF, e.g:"
+  echo -e "$ bsub -o copy_release.log -e copy_release.err $0 <input_folder> <output_folder> <intermediate_folder> <unmapped_variants_folder>\n"
+  exit 1
+fi
+
+INPUT_FOLDER=$1
+OUTPUT_FOLDER=$2
+INTERMEDIATE_FOLDER=$3
+UNMAPPED_VARIANTS_FOLDER=$4
+
+if [ ! -d ${OUTPUT_FOLDER} ]
+then
+  echo "Make sure that the output folder ${OUTPUT_FOLDER} exists. Aborting script, nothing was written."
+  exit 1
+fi
+
+mkdir ${INTERMEDIATE_FOLDER}
+if [ $? -eq 1 ]
+then
+  echo "Could not create intermediate folder ${INTERMEDIATE_FOLDER}. Aborting script, nothing was written."
+  exit 1
+fi
+
+# taken from vrnevadev:
+#
+# select distinct rs.assembly_accession, p.tax_id
+# from dbsnp_ensembl_species.rs_release_progress rs
+# join dbsnp_ensembl_species.import_progress p
+# on rs.dbsnp_db_name = p.database_name
+# where assembly_accession <> ''
+# order by assembly_accession, tax_id;
+echo "GCA_000001215.4	7227
+GCA_000001515.4	9598
+GCA_000001545.3	9601
+GCA_000001635.4	10090
+GCA_000001635.5	10090
+GCA_000001635.6	10090
+GCA_000001735.1	3702
+GCA_000001895.4	10116
+GCA_000002035.2	7955
+GCA_000002035.3	7955
+GCA_000002175.2	9598
+GCA_000002195.1	7460
+GCA_000002265.1	10116
+GCA_000002275.2	9258
+GCA_000002285.1	9615
+GCA_000002285.2	9615
+GCA_000002295.1	13616
+GCA_000002305.1	9796
+GCA_000002315.3	9031
+GCA_000002335.3	7070
+GCA_000002655.1	330879
+GCA_000002765.1	5833
+GCA_000002775.1	3694
+GCA_000002985.3	6239
+GCA_000003025.4	9823
+GCA_000003025.6	9823
+GCA_000003055.5	9913
+GCA_000003195.1	4558
+GCA_000003625.1	9986
+GCA_000003745.2	29760
+GCA_000004515.2	3847
+GCA_000004515.3	3847
+GCA_000004665.1	9483
+GCA_000005005.6	4577
+GCA_000005425.2	4530
+GCA_000005575.1	7165
+GCA_000146605.3	9103
+GCA_000146795.3	61853
+GCA_000148765.2	3750
+GCA_000151805.2	59729
+GCA_000151905.1	9593
+GCA_000181335.3	9685
+GCA_000188115.2	4081
+GCA_000188235.2	8128
+GCA_000219495.1	3880
+GCA_000224145.2	7719
+GCA_000233375.4	8030
+GCA_000237925.2	6183
+GCA_000247795.2	9915
+GCA_000247815.2	59894
+GCA_000298735.1	469796
+GCA_000298735.2	9940
+GCA_000309985.1	3711
+GCA_000317375.1	79684
+GCA_000317765.1	9923
+GCA_000317765.1	9925
+GCA_000331145.1	3827
+GCA_000355885.1	8839
+GCA_000364345.1	9541
+GCA_000372685.1	7994
+GCA_000409795.2	60711
+GCA_000413155.1	42345
+GCA_000442705.1	51953
+GCA_000686985.1	3708
+GCA_000695525.1	3712
+GCA_000710875.1	4072
+GCA_000772875.3	9544
+GCA_000966335.1	7950
+GCA_000987745.1	3635
+GCA_001433935.1	4530
+GCA_001465895.2	105023
+GCA_001522545.1	9157
+GCA_001577835.1	93934
+GCA_001625215.1	79200" > /tmp/assembly_to_taxonomy_map.txt
```

Comment:
RHEL, CentOS (systemd-tmpfiles-clean.service) and other enterprise distributions have daemons that periodically clean out /tmp and it is better not to rely on it unless absolutely unavoidable.